blackjax/__init__.py

```diff
@@ -33,4 +34,5 @@
     "ess",  # diagnostics
     "rhat",
     "pathfinder_adaptation",
+    "latent_gaussian",
```

Comment:
`latent_gaussian` refers to a class of models rather than a specific sampler. We should find a better name imo.

Reply:
The paper calls it mGrad for "marginal gradient sampler for latent Gaussian models". I was trying a few things but it's a bit of a mouthful...